src/daemon/daemon.cpp

```diff
@@ -1709,6 +1717,7 @@ grpc::Status mp::Daemon::version(grpc::ServerContext* context, const VersionRequ
 
     VersionReply reply;
     reply.set_version(multipass::version_string);
+    config->update_prompt->populate(reply.mutable_update_info());
     server->Write(reply);
     return grpc::Status::OK;
 }
```

Comment:
So on the daemon side I only see changes in `list`, `start`, and `version`. But `launch` was modified elsewhere. Did you forget it here?

Reply:
Seems like `launch` in the daemon was forgotten to me as well since `launch` in the client is equipped to handle this.

Reply:
done